export/tests/print/test_service.py

```diff
@@ -325,41 +329,58 @@ def test__print_all_files(self, mock_wait):
         self.service._print_all_files()
         mock_print.assert_has_calls(
             [
-                mock.call(f"{self.submission.tmpdir}/export_data/file1.txt"),
-                mock.call(f"{self.submission.tmpdir}/export_data/file2.txt"),
-                mock.call(f"{self.submission.tmpdir}/export_data/file3.txt"),
+                mock.call(Path(self.submission.tmpdir, "export_data/file1.txt")),
+                mock.call(Path(self.submission.tmpdir, "export_data/file2.txt")),
+                mock.call(Path(self.submission.tmpdir, "export_data/file3.txt")),
             ],
             any_order=True,
         )
         p.stop()
 
     @mock.patch("securedrop_export.print.service.Service._wait_for_print")
-    def test_open_office_file_convert_to_pdf(self, mock_wait):
-        file = "/tmp/definitely-an-office-file.odt"
+    def test__print_file_odt_calls_libreoffice_conversion_then_print(self, mock_wait):
+        file = Path("/tmp/export-data/office-file.odt")
 
         with (
-            mock.patch("subprocess.check_call") as sp,
+            mock.patch("subprocess.check_call") as check_call,
             mock.patch("securedrop_export.print.service.logger.info") as log,
+            mock.patch(
+                "subprocess.check_output",
+                side_effect=[
+                    b"/tmp/export-data/office-file.odt: application/vnd.oasis.opendocument.text\n",
+                    b"Convert office-file.odt -> office-file.pdf\n",
+                ],
+            ) as check_output,
+            mock.patch("Path.exists", return_value=True),
```

Comment:
This is failing in CI because it should be mocking `pathlib.Path.exists`, but then it fails because there are other Path.exists() calls that get invoked and some of them should return False (e.g. checking if the PDF file already exists).